hdf/src/dfstubs.c

```diff
@@ -48,46 +48,6 @@
-            return (ret);                                                                                    \
-        }                                                                                                    \
-    }
-
-#define CKSEEK(x, y, z, ret)                                                                                 \
-    {                                                                                                        \
-        if (DF_SEEK(x, (long)y, z) < 0) {                                                                    \
-            DFerror = DFE_SEEKERROR;                                                                         \
-            return (ret);                                                                                    \
-        }                                                                                                    \
-    }
-
-#define CKSEEKEND(x, y, z, ret)                                                                              \
-    {                                                                                                        \
-        if (DF_SKEND(x, (long)y, z) < 0) {                                                                   \
-            DFerror = DFE_SEEKERROR;                                                                         \
-            return (ret);                                                                                    \
-        }                                                                                                    \
-    }
-
-#define CKREAD(x, y, z, f, ret)                                                                              \
-    {                                                                                                        \
-        if (DF_READ((char *)x, (int)(y), (int)(z), (f)) < 0) {                                               \
-            DFerror = DFE_READERROR;                                                                         \
-            return (ret);                                                                                    \
-        }                                                                                                    \
-    }
-
-#define CKWRITE(x, y, z, f, ret)                                                                             \
-    {                                                                                                        \
-        if (DF_WRITE((char *)x, (int)y, (int)z, f) < 0) {                                                    \
-            DFerror = DFE_WRITEERROR;                                                                        \
-            return (ret);                                                                                    \
-        }                                                                                                    \
-    }
-
```

Comment:
These are also unused and invoke the dfi.h macros. Since they are located in a C file, there is no risk if we remove them.